### PR TITLE
LPS-133435 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133435
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Portlet.path [hunk 1]
```diff
@@ -18,6 +18,26 @@
...
+<tr>
+	<td>BORDER_STYLE_WIDTH_INPUT</td>
+	<td>//div[contains(@class,'border-width')]//input[contains(@id,'${key_fieldLabel}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>BORDER_STYLE_WIDTH_SELECT</td>
+	<td>//div[contains(@class,'border-width')]//select[contains(@id,'${key_fieldLabel}')]</td>
+	<td></td>
+</tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Portlet.path:40]: Move to Select.path file then rename a general one
ANY_ID
//select[contains(@id,'${key_value}')]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase [hunk 2]
```diff
@@ -1168,6 +1168,68 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
...
+				groupName = "Test Site Name",
+				layoutName = "Test Page Name");
+		}
+
+		task ("Add a Web Content Display to page via Widgets tab of Add panel") {
+			Navigator.gotoSitePage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			Portlet.addPG(portletName = "Web Content Display");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase:1185]: Use json macro

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase [hunk 3]
```diff
@@ -1168,6 +1168,68 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
 			userEmailAddress = "test@liferay.com");
 	}
 
+	@description = "This is a test for LPS-129438. View the changes of Border Styles in WCD on widget page."
+	@priority = "4"
+	test ViewChangesOfBorderStylesInWCDOnWidgetPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase:1173]: ConfigureLookAndFeelOfWCD

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update the cases according to comments?

**Comment by ozysouza** [comment]
ci:open

**Comment by ozysouza** [comment]
Hi @Tim-Cao PR updated following your comments.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase [hunk 4]
```diff
@@ -584,6 +584,58 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
...
+
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-color",
+				value1 = "rgba(0, 0, 0, 0)");
+
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-width",
+				value1 = "1px");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase:603]: View border-top-width, border-left-width, border-right-width and border-bottom-width

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase [hunk 5]
```diff
@@ -584,6 +584,58 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
...
+	@priority = "4"
+	test ConfigureLookAndFeelBorderStyleOfWCD {
+		task ("Assert the default value in border style on WCD") {
+			Navigator.gotoSitePage(
+				pageName = "Web Content Display Page",
+				siteName = "Test Site Name");
+
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-color",
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase:597]: View border-top-color, border-left-color, border-right-color and border-bottom-color

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase [hunk 6]
```diff
@@ -584,6 +584,58 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
...
+
+			IFrame.closeFrame();
+
+			Refresh();
+		}
+
+		task ("Assert the new border style is applied to WCD") {
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-color",
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase:629]: View border-top-color, border-left-color, border-right-color and border-bottom-color

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase [hunk 7]
```diff
@@ -584,6 +584,58 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
...
+				value1 = "rgb(255, 13, 13)");
+
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-width",
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase:634]: View border-top-width, border-left-width, border-right-width and border-bottom-width

**Comment by Tim-Cao** [comment]
Hi @ozysouza, could you update again?

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you update again?

@Tim-Cao  Of course, PR updated.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/4689131693cd85c44473c4a142f02bf1eaec7c9a...10e4838da69dd80ac04e26433ada3a9c57b87e69
in https://github.com/liferay-echo/liferay-portal/pull/5052

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**